src/ucm/mmap/install.c

```diff
@@ -86,8 +87,7 @@ void ucm_fire_mmap_events(int events)
     }
 
     if (events & (UCM_EVENT_SBRK|UCM_EVENT_VM_MAPPED|UCM_EVENT_VM_UNMAPPED)) {
-        (void)sbrk(ucm_get_page_size());
-        (void)sbrk(-ucm_get_page_size());
+        (void)sbrk(0);
```

Comment:
sbrk(0) will not produce vm_map/vm_unmap events

Reply:
i'm relying on other tests to provide it.. 
however i could probably update the 'if' and add comment to document this